### PR TITLE
DFRN comments are now stored as comments again

### PR DESCRIPTION
**Pull request by annando**
Additionally to fixing this error, this PR also addresses the issue of some notice.

### CODE REVIEW

File: src/Protocol/DFRN.php [hunk 1]
```diff
@@ -2603,8 +2603,8 @@ private static function processEntry($header, $xpath, $entry, $importer, $xml)
 		$item["parent-uri"] = $item["uri"];
 
 		$inreplyto = $xpath->query("thr:in-reply-to", $entry);
-		if (!empty($inreplyto->item[0])) {
-			foreach ($inreplyto->item[0]->attributes as $attributes) {
+		if (is_object($inreplyto->item(0))) {
+			foreach ($inreplyto->item(0)->attributes as $attributes) {
```
Comment (MrPetovan) [inline src/Protocol/DFRN.php]: Wait, what is this notation?

Comment (annando) [reply src/Protocol/DFRN.php]: ```item``` is no array, it is an object that you address via the integer value. It is done at many places all over the system - and is working fine.

Comment (MrPetovan) [reply src/Protocol/DFRN.php]: I still don't understand, is it a method invocation? A property retrieval?

Comment (annando) [reply src/Protocol/DFRN.php]: I must confess, I don't know. But I know that it works ;-)

Comment (MrPetovan) [reply src/Protocol/DFRN.php]: *sigh*

Comment (MrPetovan) [reply src/Protocol/DFRN.php]: After a cursory research, it actually is a method: http://php.net/manual/en/domnodelist.item.php

Comment (JeroenED) [reply src/Protocol/DFRN.php]: It's a method invocation. But its return value implement interface traversable to be accessed as array. https://secure.php.net/manual/en/class.traversable.php

Comment (annando) [reply src/Protocol/DFRN.php]: @MrPetovan I just wanted to post that link as well.

Comment (MrPetovan) [reply src/Protocol/DFRN.php]: 🙌

(my girlfriend is getting suspicious about us two)

**Pull request merged**